README.md

```diff
@@ -95,7 +95,8 @@ CMake invocation with
 build/$ cmake -DBUILD_TESTING=True -DHSM_ENABLED=ON ..
 ```
 
-[libp11](https://github.com/OpenSC/libp11/) `>=libp11-0.4.12` is required for building MoCOCrW with
+[libp11 release 0.4.12](https://github.com/OpenSC/libp11/releases/tag/libp11-0.4.12) patched with
+[patch for key generation](https://github.com/bmwcarit/MoCOCrW/blob/openssl1.1/dockerfiles/feature-support/hsm-patches/0001-Introduce-generic-keypair-generation-interface-and-e.patch) is required for building MoCOCrW with
```

Comment:
Add link to build instructions for libp11? Users would need to know this information to build along with your patch.

Reply:
Done